### ISSUE TITLE
前后两个版本算法方面

### ISSUE THREAD

**Issue by yyaqi** [YunYang1994/tensorflow-yolov3#144]
您好，请问前后两个版本的代码在算法方面有改动吗

**Comment by YunYang1994** [comment]
差别不大，第一版本是用的tf.contrib.slim框架，但是这个框架已经deprecated了，所以重新更新了下代码。另外使用了最新的giou loss function，训练效果提高了些

**Comment by yyaqi** [comment]
> 差别不大，第一版本是用的tf.contrib.slim框架，但是这个框架已经deprecated了，所以重新更新了下代码。另外使用了最新的giou loss function，训练效果提高了些

好的，谢谢

**Comment by YunYang1994** [comment]
> > 差别不大，第一版本是用的tf.contrib.slim框架，但是这个框架已经deprecated了，所以重新更新了下代码。另外使用了最新的giou loss function，训练效果提高了些
> 
> 好的，谢谢

最好重新git clone下吧！

**Comment by yyaqi** [comment]
> > > 差别不大，第一版本是用的tf.contrib.slim框架，但是这个框架已经deprecated了，所以重新更新了下代码。另外使用了最新的giou loss function，训练效果提高了些
> > 
> > 
> > 好的，谢谢
> 
> 最好重新git clone下吧！

嗯嗯，好的，谢谢
